frontends/PyCDE/test/test_hwarith.py

```diff
@@ -57,16 +57,24 @@ def construct(ports):
 # CHECK: msft.module @SignlessInfixComparison {} (%in0: i16, %in1: i16)
 # CHECK-NEXT:    %0 = comb.icmp bin eq %in0, %in1 {{({sv.namehint = ".*"} )?}}: i16
 # CHECK-NEXT:    %1 = comb.icmp bin ne %in0, %in1 {{({sv.namehint = ".*"} )?}}: i16
+# CHECK-NEXT:    %1 = comb.icmp bin lt %in0, %in1 {{({sv.namehint = ".*"} )?}}: i16
+# CHECK-NEXT:    %1 = comb.icmp bin gt %in0, %in1 {{({sv.namehint = ".*"} )?}}: i16
+# CHECK-NEXT:    %1 = comb.icmp bin le %in0, %in1 {{({sv.namehint = ".*"} )?}}: i16
+# CHECK-NEXT:    %1 = comb.icmp bin ge %in0, %in1 {{({sv.namehint = ".*"} )?}}: i16
 # CHECK-NEXT:    msft.output
 @unittestmodule(run_passes=True)
 class SignlessInfixComparison(Module):
-  in0 = Input(types.i16)
-  in1 = Input(types.i16)
+  in0 = Input(SInt(16))
```

Comment:
This test explicitly tests signless ints. By changing the type, you change the test. Plz remove this change.